### PR TITLE
don't re-deploy on domains:add. allow multple domains on command line. closes #1142

### LINKED ISSUE #1142: crash adding a domain
```
dokku domains:add core-server-staging get.v1.staging.core
-----> Releasing core-server-staging ...
runtime: panic before malloc heap initialized
fatal error: runtime: cannot allocate heap metadata
```

Then I realized the hostname was wrong, then I tried again with a proper full name:

```
dokku domains:add core-server-staging get.v1.staging.core.namoroon.com
-----> Releasing core-server-staging ...
runtime: panic before malloc heap initialized
fatal error: runtime: cannot allocate heap metadata
```

And now? How can I move forward and have domains added? Is there a workaround? Thanks.

### PR DESCRIPTION
**Pull request by michaelshobbs**

### CODE REVIEW

**Comment by josegonzalez** [comment]
No test for multiple domains? For shame.

File: tests/unit/domains.bats [hunk 1]
```diff
@@ -28,6 +28,13 @@ teardown() {
   assert_success
 }
 
+@test "(domains) domains:add (multiple)" {
+  run dokku domains:add $TEST_APP www.test.app.dokku.me test.app.dokku.me
```
Comment (michaelshobbs) [inline tests/unit/domains.bats]: this tests multiple domains....

Comment (josegonzalez) [inline tests/unit/domains.bats]: Thats gotta be new. Or I can't read...

**Comment by josegonzalez** [comment]
:)

**Comment by michaelshobbs** [comment]
Haha!

**Pull request merged**